src/API/IPFSConfiguration.ts

```diff
@@ -0,0 +1,5 @@
+/* tslint:disable:interface-name */
+
+export interface IPFSConfiguration {
```

Comment:
Please move this interface into the class file because it belongs to the class. We are no longer separating class configuration interfaces from the class.